extern/ProtoGeometry/ProtoGeometry.config

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0"?>
 <ProtoGeometryConfiguration xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-  <GeometryFactoryFileName>libg_219/LibG.ProtoInterface.dll</GeometryFactoryFileName>
+  <GeometryFactoryFileName>libg_220/LibG.ProtoInterface.dll</GeometryFactoryFileName>
```

Comment:
@pboyer Yeah. Sorry. That is in there because I made the change on this branch, but actually issued a separate PR from master (for you to review). I can remove those commits here.